opendevin/server/listen.py

```diff
@@ -134,8 +150,14 @@ def select_file(file: str):
     except Exception as e:
         logger.error(f'Error opening file {file}: {e}', exc_info=False)
         error_msg = f'Error opening file: {e}'
-        return Response(f'{{"error": "{error_msg}"}}', status_code=500)
-    return {'code': content}
+        return JSONResponse(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            content={'error': error_msg},
+        )
+    return JSONResponse(
+        status_code=status.HTTP_200_OK,
```

Comment:
I'm going to guess that `200` is the default, so maybe we don't need to specify these?

Reply:
It's true that many web frameworks default to a `200` OK status for successful HTTP responses, explicitly setting the status code can have benefits for code consistency and clarity

Reply:
Being more concise seems more pythonic to me.

Reply:
So maybe for `200` status code response, just return the `content` without `JSONResponse()`  and `status_code` specified. Like this:

```
@app.get('/api/refresh-files')
def refresh_files():
    structure = files.get_folder_structure(Path(str(config.get('WORKSPACE_BASE'))))
    return structure.to_dict()


@app.get('/api/select-file')
def select_file(file: str):
    try:
        # xxx
    except Exception as e:
        # yyy
        return JSONResponse(
            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
            content={'error': error_msg},
        )
    return {'code': content}


```

Reply:
👍 I like that